types/kavenegar/tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "exactOptionalPropertyTypes": true
```

```suggestion
        "forceConsistentCasingInFileNames": true
```
No, we do not want this, we want the typescript code details to be optionally compatible with this option, if someone uses this in consuming code (say you're writing an app with strict mode upped up in ts config)

---

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54253

---

I understand what you mean now. Yes I agree with you. I fixed it in my last commit.